src/capi/azext_capi/helpers/generic.py

```diff
@@ -7,10 +7,15 @@
 This module contains helper functions for the az capi extension.
 """
 
+# pylint: disable=missing-docstring
```

Comment:
Can we just add a docstring to whatever function needs it instead of disabling the linter?